### DISCUSSION TITLE
Need to know details on how I can setup the plugin perfectly?

### DISCUSSION THREAD

**Discussion by tarikul47** [jamieblomerus/WP-Mobile-BankID-Integration#30] [Support]
Hello! 
I hope to find you well. You are doing great work! Thank you very much! 

By the way, I installed and activated it on my site. I can't configure it properly.

1). From what I've read in the BankID Developer documentation, I need a certificate and a password. I also confirm that after enabling the plugin I found that for production mode there were two input fields one for certificate and one for password ( https://prnt.sc/h8-KBfUSgvhf ). I hope that if I apply to a particular bank they will give me the certificate and password, if I upload and save it in this input of this plugin, our login system will work properly. Please confirm if I understood correctly. 

2). Before I go into production mode I wanted to check with my site to be a little practical in testing mode. That's why when I activated this plugin, I activated the auto-testing mode from the settings of this plugin [ https://prnt.sc/3mMULRXfStSh ]

Then I checked it practically on WordPress's default login page where there is a button Login with Bank Account ID and after clicking it I can see it is not working properly and there is no QR code being generated. 

Receiving orderRef and autoStartToken = null ( https://prnt.sc/2x6eeY-isCZU )

I will be very happy and very grateful to you if you let me know where I am doing wrong or how I can practically check by activating the testing mode and in my production it will work as I said. 

Thank you very much for creating such a nice plugin it will be really helpful for the community inshallah.

**Comment by jamieblomerus** [comment]
Hello @tarikul47!
Thank you for your kind words and for using the plugin. I’m happy to help you with the configuration.

### 1. Production Mode Configuration:

Yes, your understanding is correct. To use BankID in production mode:

1. **Obtain a Production Certificate and Password:** Apply through a bank to receive the certificate and password.
2. **Upload Certificate and Enter Password:** In the plugin’s setup wizard under production mode, specify the file path of the uploaded certificate and enter its password.
3. **Save and Test:** After saving these settings, your login system should work with BankID in production mode.

### 2. Testing Mode Troubleshooting:

For the issues in testing mode, I’ll need a bit more information to identify the problem. Could you please enable WP_DEBUG on the site or, if possible, provide temporary login details?

Best regards,
Jamie

**Comment by jamieblomerus** [comment]
Hello @tarikul47,

Never mind—I’ve just managed to reproduce the issue on my end! I’ll investigate further and keep you updated.

Best regards,
Jamie

**Comment by jamieblomerus** [comment]
Hello @tarikul47,

Good news—the issue has been fixed in the latest release, version 1.4! Please update to this version, and let me know if everything works smoothly on your end.

Also, if you found the plugin helpful, I’d really appreciate it if you could star the GitHub repo and mark this message as the answer.

Thanks again for your support!

Best regards,
Jamie

**Comment by tarikul47** [comment]
I am really sorry for not being able to reply to you due to my busy
schedule. Alhamdulillah after updating the plugin the testing mode was
working fine. I pray for you and your family, and may Allah bless you.
Thank you so much.
Best regards,
Tarikul Islam
